ui/src/app/cohort-search/list-search-v2/list-search-v2.component.tsx

```diff
@@ -377,11 +390,15 @@ export const ListSearchV2 = fp.flow(withCdrVersions(), withCurrentWorkspace())(
             <table className='p-datatable' style={styles.table}>
               <thead className='p-datatable-thead'>
                 <tr style={{height: '2rem'}}>
-                  <th style={{...styles.columnHeader, borderLeft: 0}}>Name</th>
-                  <th style={{...styles.columnHeader, width: '20%'}}>Code</th>
-                  <th style={{...styles.columnHeader, width: '10%'}}>Vocab</th>
-                  <th style={{...styles.columnHeader, width: '8%'}}>Count</th>
-                  <th style={{...styles.columnHeader, textAlign: 'center', width: '12%'}}>View Hierarchy</th>
+                  <th style={{...styles.columnNameHeader, width: '31%', borderLeft: 0}}>Name</th>
+                  <th style={columnHeaderStyle}>Code</th>
+                  <th style={columnHeaderStyle}>Vocab</th>
+                  <th style={columnHeaderStyle}>Count</th>
```

Comment:
May need to verify with the team, but I think the old count column should be removed once we add the new count columns.

Reply:
Done